test/unit/view_model/record_test.rb

```diff
@@ -338,10 +392,25 @@ def new_record?
       end
     end
 
+    describe 'with unspecified attributes falling back to the model default' do
+      let(:attributes)           { { value: {} } }
+      let(:default_view_values)  { { } }
```

Comment:
To be consistent with my understanding of `default_view_values`, this should be expressed as `let(:default_values) { { value: 5 } }`, and the deserialize should use `deserialize_from_view({}, ...)`

Reply:
I addressed this slightly differently, by separating the test subjects from the defaults.